bin/asd-prereq-checker.sh

```diff
@@ -16,6 +16,16 @@ DISABLED_SERVICE_MANAGEMENT_API="{
   'Message': 'Service Management API is not enabled. You must enable this API in the current project. https://console.cloud.google.com/apis/api/servicemanagement.googleapis.com/overview?project=$PROJECT_ID '
 }"
 
+INVALID_ORG_POLICY_OSLOGIN="{
+  'KnownIssueId': 'invalid_org_policy',
```

Comment:
Did we want to make the invalid_org_policy distinct (i.e. invalid_org_policy_os_login) or are we good with just having a generic ID?

Reply:
Good question, I wasn't sure how much granularity we need here.  From an instrumentation perspective, would you recommend that we do?

Reply:
It might be easier to see a breakdown of which org policies are causing the most issues for users.

Reply:
Done.